packages/babel-parser/test/fixtures/es2015/generators/yield-regex/input.js

```diff
@@ -0,0 +1 @@
+function* x() { yield y * /z/ }
```

Comment:
In this test we are not actually yielding a regexp. Did you mean something like `yield* /z/`?

Reply:
yeah sorry